Assignment2/marioruiz2003-MarioRuiz.ttl

```diff
@@ -0,0 +1,17 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix : <http://example.org/#> .
+
+:Class01  rdfs:includes :Sensor029  ;
+   rdfs:includes :Computer101
+.
+
+:Sensor029 rdfs:hasMeasurement :Measurement8401.
+
+:Computer101 rdfs:hasOwner :User10A;
+   
+.
+:Measurement8401 rdfs:hasTemperature "29";
+     rdfs:atTime "2010-06-12T12:00:12".
```

Comment:
Wrong use of rdfs prefix, and you should add a datatype to the strings hereand in the previous triple.